src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -0,0 +1,230 @@
+                                        <FormattedMessage {...messages.annotationActivityDeleteMenuItem} />
+                                    </MenuItem>
+                                )}
+                            </Media.Menu>
+                            {isConfirmingDelete && (
+                                <DeleteConfirmation
+                                    data-resin-component={ACTIVITY_TARGETS.COMMENT_OPTIONS}
+                                    isOpen={isConfirmingDelete}
+                                    message={messages.annotationActivityDeletePrompt}
+                                    onDeleteCancel={handleDeleteCancel}
+                                    onDeleteConfirm={handleDeleteConfirm}
+                                />
+                            )}
+                        </TetherComponent>
+                    )}
+                    <div className="bcs-AnnotationActivity-headline">
+                        <UserLink
+                            data-resin-target={ACTIVITY_TARGETS.PROFILE}
+                            id={createdByUser.id}
+                            name={createdByUser.name}
+                            getUserProfileUrl={getUserProfileUrl}
+                        />
+                    </div>
+                    <div>
+                        <ActivityTimestamp date={createdAtTimestamp} />
+                    </div>
+                    {isEditing ? (
+                        <CommentForm
+                            isDisabled={isDisabled}
+                            className={classNames('bcs-AnnotationActivity-editor', {
+                                'bcs-is-disabled': isDisabled,
+                            })}
+                            updateComment={handleUpdate}
+                            isOpen={isInputOpen}
+                            user={currentUser}
+                            onCancel={commentFormCancelHandler}
+                            onFocus={commentFormFocusHandler}
+                            isEditing={isEditing}
+                            entityId={id}
+                            showTip={false}
+                            tagged_message={tagged_message}
+                            getAvatarUrl={getAvatarUrl}
+                            mentionSelectorContacts={mentionSelectorContacts}
+                            getMentionWithQuery={getMentionWithQuery}
+                        />
+                    ) : (
+                        <ActivityMessage
+                            id={id}
+                            tagged_message={tagged_message}
+                            translatedTaggedMessage={translatedTaggedMessage}
+                            {...translations}
+                            translationFailed={error ? true : null}
+                            getUserProfileUrl={getUserProfileUrl}
+                        />
+                    )}
+                    <AnnotationActivityItem
```

Comment:
Still working through this for deciding if the annotation is a current version annotation or past version